dashboard/test/ui/features/foundations/hamburger.feature

```diff
@@ -2,19 +2,56 @@
 @single_session
 Feature: Hamburger dropdown
 
-  Scenario: Signed out user in English should not see hamburger on desktop
+  Scenario: Signed out user in English should not see hamburger on large desktop
     Given I am on "http://code.org/"
     And I dismiss the language selector
     Then I wait to see ".header_button"
     Then element "#hamburger-icon" is not visible
 
+  Scenario: Signed out user in English should see hamburger on small desktop
+    Given I am on "http://code.org/"
+    And I dismiss the language selector
+    And I change the browser window size to 1268 by 768
```

Comment:
TIL you can set browser size in UI tests!

Reply:
Isn't this great! It's set up here: https://github.com/code-dot-org/code-dot-org/blob/133b02f34a09c8bf208fb380268144ae6953eea9/dashboard/test/ui/features/step_definitions/browser_control.rb#L1-L3